a-braham/Questioner
# Bg view question 163347911

#### What does this PR do?
 * The pull request modifies view_question.html and comment.html page

#### Description of Task to be completed?
  **Have the following features complete**
  * The comment form should be simple
  * Pages should be responsive
  * Should not have any inline css
  * A user should not be able to comment if not logged in

#### How should this be manually tested?
  * Clone the repository
  * Open the /view_question.html and comment.html files with chrome
  * Test for responsiveness, check the html tags and css for errors
 
#### What are the relevant pivotal tracker stories?
  #163347911

#### Screenshots
![screenshot from 2019-01-20 12-49-23](https://user-images.githubusercontent.com/45788436/51437637-f2cfbb00-1cb1-11e9-8325-e010e4e12479.png)
![screenshot from 2019-01-20 12-49-50](https://user-images.githubusercontent.com/45788436/51437638-f2cfbb00-1cb1-11e9-9247-4f7575b88efb.png)
